package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lob/openapi",
-  "version": "1.19.24",
+  "version": "1.19.25",
```

Comment:
do we need to upgrade package-lock.json too?

Reply:
Yeah probably. I wasn't sure about committing the package-lock.json too. I'll push it.